modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro

```diff
@@ -40,6 +40,8 @@ definition {
 	macro addPicklistItemViaUI {
 		SelectFrame(locator1 = "Picklist#IFRAME_PICKLIST_MODAL");
 
+		WaitForPageLoad();
+
 		Click.javaScriptClick(locator1 = "Picklist#ADD_ITEM_BUTTON");
 
```

Comment:
In the future, please add this in a separate commit. It's unclear why we need `WaitForPageLoad();` here